test/Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests/HttpsTest.cs

```diff
@@ -111,7 +119,7 @@ public Task Https_HelloWorld_NoClientCert_CoreCLR(RuntimeFlavor runtimeFlavor, R
         [ConditionalTheory(Skip = "Manual test only, selecting a client cert is non-determanistic on different machines.")]
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
         [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR)]
-        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, "https://localhost:44401/", ApplicationType.Portable)]
+        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, "https://localhost:44301/", ApplicationType.Portable)]
```

Comment:
What @Tratcher said. There's a comment at the top of the class, which I can clarify. SSL bindings require pre-registering with http.sys, which includes specifying a port. Conveniently, IIS Express does this for us automatically for the `443xx` range of ports. We could use a random port allocation strategy here, or build some custom code to register the SSL binding, but it would be a lot of overhead.